lib/cb/config.rb

```diff
@@ -47,6 +47,7 @@ def set_default_api_uris
       @uri_employee_types                 ||= '/v1/employeetypes'
       @uri_company_find                   ||= '/Employer/CompanyDetails'
       @uri_job_search                     ||= '/v1/JobSearch'
+      @uri_job_search_v3                  ||= '/jobsearch/jobs'
```

Comment:
:+1: you know a timeline for that i.e. should we hold off on the merge till then?